src/redis/td_malloc.h

```diff
@@ -0,0 +1,21 @@
+/**
+ * Adaptive histogram based on something like streaming k-means crossed with Q-digest.
+ * The implementation is a direct descendent of MergingDigest
+ * https://github.com/tdunning/t-digest/
+ *
+ * Copyright (c) 2021 Redis, All rights reserved.
```

Comment:
I think it should be ok to leave this here

Reply:
it's appache 2.0 so we should be ok using it

Reply:
Yes, please keep the headers, also please add the license file [LICENSE.md](https://github.com/RedisBloom/t-digest-c/blob/master/LICENSE.md) and name it t-digest.LICENSE.md